include/vcpkg/input.h

```diff
@@ -6,17 +6,26 @@
 
 namespace vcpkg
 {
-    PackageSpec check_and_get_package_spec(std::string&& spec_string,
+    // Parse a package spec without features; typically used by commands which
+    // refer to already installed packages which make features irrelevant.
+    //
+    // Does not assert that the package spec has a valid triplet. This allows
+    // such commands to refer to entities that were installed with an overlay
+    // triplet or similar which is no longer active.
+    PackageSpec check_and_get_package_spec(StringView spec_string,
```

Comment:
It bothers me that `check_and_get_package_spec()` and `check_and_get_full_package_spec()` now have similar names but different behavior. What about renaming this function to `get_package_spec()` to further distance them?